app/controllers/checksum_records_controller.rb

```diff
@@ -26,21 +12,7 @@ def create
     end
   end
 
-  # PATCH/PUT /checksum_records/1
-  # PATCH/PUT /checksum_records/1.json
-  def update
-    if @checksum_record.update(checksum_record_params)
-      render :show, status: :ok, location: @checksum_record
-    else
-      render json: @checksum_record.errors, status: :unprocessable_entity
-    end
-  end
-
-  # DELETE /checksum_records/1
-  # DELETE /checksum_records/1.json
-  def destroy
-    @checksum_record.destroy
-  end
+  def show; end
```

Comment:
ew - same line?  I would put a `# TODO: to be implemented` for the show and the create method, maybe.